src/game/MainMenuScreen.cc

```diff
@@ -338,7 +330,6 @@ static void RenderMainMenu(void)
 {
 	BltVideoObject(FRAME_BUFFER, guiMainMenuBackGroundImage, 0, STD_SCREEN_X,       STD_SCREEN_Y     );
 	BltVideoObject(FRAME_BUFFER, guiJa2LogoImage,            0, STD_SCREEN_X + 188, STD_SCREEN_Y + 15);
-	BltVideoSurface(guiSAVEBUFFER, FRAME_BUFFER, 0, 0, NULL);
```

Comment:
Was this supposed to be in the previous commit?

Reply:
Yes, I just had overlooked it. Copying the frame buffer to the save buffer is only necessary when we must use `RestoreExternBackgroundRect()` to restore parts of the frame buffer later. The main screen however is so simple (just two images and five quick buttons) that there isn't anything in the frame buffer that needs to be saved.